central/vuln-mgmt/service/service.go

```diff
@@ -0,0 +1,27 @@
+package service
```

Comment:
On the package name: given our current styling w.r.t package names let's do `vulnmgmt` instead - we don't really have hyphens / underscores established as package names so let's not start with it now.